From 37a0c90d8a7e69cedf94cee6e84eafd945ffd02f Mon Sep 17 00:00:00 2001
From: Dima Krasner <dima@dimakrasner.com>
Date: Sun, 20 May 2012 15:47:59 +0100
Subject: [PATCH] Set a default size for the log window

The bug is just a small annoyance with the log window - it gets squashed
because no size (or automatic expanding) is specified.
---
 ROX-Filer/Templates.ui |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/ROX-Filer/Templates.ui b/ROX-Filer/Templates.ui
index dc68801..9c492b8 100644
--- a/ROX-Filer/Templates.ui
+++ b/ROX-Filer/Templates.ui
@@ -2,6 +2,8 @@
 <interface>
   <!-- interface-requires gtk+ 2.12 -->
   <object class="GtkDialog" id="Log viewer">
+    <property name="width_request">640</property>
+    <property name="height_request">300</property>
     <property name="visible">True</property>
     <property name="can_focus">False</property>
     <property name="title" translatable="yes">ROX-Filer log viewer</property>
-- 
1.6.5.GIT

